docs/fsharp/language-reference/functions/recursive-functions-the-rec-keyword.md

```diff
@@ -26,15 +26,42 @@ function2-body
 
 Recursive functions, functions that call themselves, are identified explicitly in the F# language. This makes the identifer that is being defined available in the scope of the function.
 
-The following code illustrates a recursive function that computes the *n*<sup>th</sup> Fibonacci number.
+The following code illustrates a trivial recursive function that computes the *n*<sup>th</sup> Fibonacci number.
 
 [!code-fsharp[Main](~/samples/snippets/fsharp/lang-ref-1/snippet4001.fs)]
 
 > [!NOTE]
-> In practice, code like that above is wasteful of memory and processor time because it involves the recomputation of previously computed values.
+> In practice, code like the previous sample is not ideal because it unecessarily recomputes values that have already been computed. This is because it is not tail recursive, which is explained further in the document.
```

Comment:
```suggestion
> In practice, code like the previous sample is not ideal because it unecessarily recomputes values that have already been computed. This is because it is not tail recursive, which is explained further in this article.
```